scripts/templates.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
It may be easier for someone coming into this script later if the name was a bit more descriptive ? Possibly something similar to 'templates_to_add' ?